tests/sentry/utils/test_integrationdocs.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from sentry.utils.integrationdocs import dump_doc, load_doc
+
+
+@pytest.mark.parametrize(
+    "path",
+    [
+        ("/"),
+        ("/.."),
+        ("//...."),
+        ("/%5c.."),
+        ("../"),
+        ("../../"),
+        ("../../../etc/passwd"),
+    ],
+)
+def test_path_traversal_attempt_on_load_doc_raises_exception(path):
+    with pytest.raises(Exception):
+        load_doc(path)
+
+
+@pytest.mark.parametrize(
+    "path",
+    [
+        ("/"),
+        ("/.."),
+        ("//...."),
+        ("/%5c.."),
+        ("../"),
+        ("../../"),
+        ("../../../etc/passwd"),
```

Comment:
the parens aren't doing anything here -- these are just parenthesized strings

parametrize has two flavors (1-arg + list of args, multi arg + list of arg tuples) -- this one is the former

Reply:
Done! Today I learned. 🙂

Reply:
if you want to learn more about parametrize I've got a video on it!  https://youtu.be/aQH7hyJn-No